### DISCUSSION TITLE
Set a dark theme in macOS Sonoma

### DISCUSSION THREAD

**Discussion by MGuerrera** [zim-desktop-wiki/zim-desktop-wiki#2623] [Q&A] [answered]
Hi everyone.
I installed Zim with Brew, but searching here and on Google I didn't find anything about a workaround to install a dark theme on macOS.
Any good man would like to help me?

**Comment by dehesselle** [comment]
Can you clarify: do you want to switch Zim into dark mode or do you want to install and use a 3rd party theme?

**Reply by MGuerrera** [reply]
Sorry, I was unclear, I mean a dark theme specific to Zim.
All the solutions I've found around concern Linux and Windows, I need one for macOS.

I'm not sure what you mean by "3rd party theme", from what I understand Zim doesn't have a native dark theme, so the solutions for Linux Windows had to do with Gtk if I'm not mistaken.

**Reply by dehesselle** [reply, marked as answer]
Of course there is a native theme, GTK on macOS is no different from Linux and Windows in that regard. See for yourself and run Zim after setting the following:
```bash
export GTK_THEME=Adwaita:dark
```

**Reply by MGuerrera** [reply]
This works wonderfully, thank you man, I've created a quick launcher app for it with Platypus using the zsh script:
```
#!/bin/zsh
export GTK_THEME=Adwaita:dark
/opt/homebrew/Cellar/zim/0.75.2/bin/zim
```

**Reply by dehesselle** [reply]
You're welcome! The next version of Zim will have a setting in the preferences  to do that.

Out of curiosity: any specific reason you're installing Zim via Homebrew instead of using the app?

**Reply by MGuerrera** [reply]
No particular reason.
I'm not very familiar with macOS but my family gave me a Mac as a gift.
I stumbled upon Homebrew and I believed that it was used to install applications in an equivalent way to the native method via dmg package, and finding Zim in its repository I didn't bother to search on the site for the availability of an app for macOS, but perhaps it would have been better to install the app.
Should I remove the installation made with brew and use the app?
The introduction of an integrated theme switching option will be a welcome change.

**Comment by dehesselle** [comment]
Closing.
